File: result/repair/transcoder/java2python/python_repair/RECURSIVELY_BREAK_NUMBER_3_PARTS_GET_MAXIMUM_SUM_1.py
def f_gold ( n ) :
    dp = [ 0 ] * ( n + 1 )
    dp [ 0 ] = 1
    for i in range ( 2 , n + 1 ) :
        dp [ i ] = max ( dp [ i // 2 ] + dp [ i // 3 ] + dp [ i // 4 ] , i )
    return dp [ n ]

File: result/translate/transcoder/python2java/java/RECURSIVELY_BREAK_NUMBER_3_PARTS_GET_MAXIMUM_SUM_1.java
public static int f_gold ( int n ) {
  int [ ] dp = new int [ n + 1 ] ;
  dp [ 0 ] = 0 ;
  dp [ 1 ] = 1 ;
  for ( int i = 2 ;
  i <= n ;
  i ++ ) {
    dp [ i ] = Math . max ( dp [ Integer . MAX_VALUE ] + dp [ Integer . MIN_VALUE ] + dp [ Integer . MIN_VALUE ] , i ) ;
    ;
  }
  return dp [ n ] ;
}
